### DISCUSSION TITLE
Is there anyway to skip pair btwn host and server?

### DISCUSSION THREAD

**Discussion by kane3345678** [LizardByte/community#180] [Sunshine] [answered]
Hi  
I'm testing sunshine and moonlight streaming.  
But there is a paring mechanism to ask pair code.  
Is there anyway to skip the paring step so that i can connect any device without doing paring on server side?  
![image](https://user-images.githubusercontent.com/2068428/220023325-a57cd1e7-5256-451d-8e9f-cce209c949e0.png)    
thank you

**Comment by ReenigneArcher** [comment, marked as answer]
No, but you can pair from a mobile device... just use your browser and connect to the sunshine web UI.

**Reply by kane3345678** [reply]
thanks for your reply!
